### ISSUE TITLE
tableview: media (images and video) should not be visible in tableview (not even in rich text fields)

### ISSUE THREAD

**Issue by simonfossom** [huridocs/uwazi#3222]
<img width="1088" alt="Screen Shot 2020-09-28 at 16 25 16" src="https://user-images.githubusercontent.com/8134850/94447575-2f9fc500-01aa-11eb-9be2-8cd145a137fd.png">

**Comment by fnocetti** [comment]
cc: @RafaPolit

**Comment by RafaPolit** [comment]
For this we would need to parse the markdown specifically for the table view.

It can be done in time, but right now it becomes a 'preview' of sorts, I think it invites you to 'click and see what is there'.

My problem is assuming that one use case is everyone's use case.  I, for one, find it useful to see in table view: "hey, here there is an image you cannot see, but know it's there".

**Comment by txau** [comment]
I don't think this is a bug, but rather a nuance. I guess some users may be interested in showing images in table view. Since we have column selection, user can opt to hide the images.

**Comment by RafaPolit** [comment]
I completely agree.

**Comment by simonfossom** [comment]
I see what you mean, but if you think about it, it's not really any sort of preview because no image is so small to be visible. Also, if user clicks one of those images it start to play video for which controls are not visible and user won't know what's happening. It's usability issue.

> My problem is assuming that one use case is everyone's use case

@RafaPolit not sure what you mean here. ☝️ 

> I don't think this is a bug, but rather a nuance. I guess some users may be interested in showing images in table view. Since we have column selection, user can opt to hide the images.

Yes, I agree. It would be useful for users to have control over showing rich-text fields in full. But it should be their choice.

BTW this is not a priority issue in my opinion but I wanted to report it for when we have time to resolve it.

**Comment by RafaPolit** [comment]
> @RafaPolit not sure what you mean here. ☝️

I meant that: assuming all table view users don't want to see pictures on rich text is assuming too much.  

Given that it is a lot of effort to create parsers that would remove those images (@fnocetti looked into already-made parsers but there were none that looked promising enough) and the fact that it is assuming too much, I'm OK with leaving that 'one line preview' of the image, even it shows nothing for the time being.

We need to find a good global solution for the rich text field.  It's not only images that are problems, just including a level 1 header yields nothing in the one-line row.

**Comment by simonfossom** [comment]
Just a note, every image should have captions (alt-text) anyway. This would be a good option to show in table
